includes/admin/class-edd-notices.php

```diff
@@ -221,6 +221,9 @@ public function show_notices() {
 					case 'email_sent' :
 						$notices['updated']['edd-payment-sent'] = __( 'The purchase receipt has been resent.', 'easy-digital-downloads' );
 						break;
+					case 'email_send_failed':
+						$notices['error']['edd-payment-sent'] = __( 'The purchase receipt was not successfully sent.', 'easy-digital-downloads' );
```

Comment:
```suggestion
						$notices['error']['edd-payment-sent'] = __( 'Failed to send purchase receipt.', 'easy-digital-downloads' );
```

What do you think about this wording instead?